content/pt-br/docs/reference/kubectl/cheatsheet.md

```diff
@@ -60,59 +58,60 @@ kubectl config view
  
-kubectl config unset users.foo                       # excluir usuário foo
+kubectl config unset users.foo                       # exclue usuário foo
 
 # alias curto para definir/mostrar contexto/namespace (funciona apenas para bash e shells compatíveis com bash, contexto atual a ser definido antes de usar kn para definir namespace)
 alias kx='f() { [ "$1" ] && kubectl config use-context $1 || kubectl config current-context ; } ; f'
 alias kn='f() { [ "$1" ] && kubectl config set-context --current --namespace $1 || kubectl config view --minify | grep namespace | cut -d" " -f6 ; } ; f'
 ```
 
-## Aplicar
+## Kubectl apply
+
 `apply` gerencia aplicações através de arquivos que definem os recursos do Kubernetes. Ele cria e atualiza recursos em um cluster através da execução `kubectl apply`.
-Esta é a maneira recomendada de gerenciar aplicações Kubernetes em ambiente de produção. Veja a [documentação do Kubectl](https://kubectl.docs.kubernetes.io).
+Esta é a maneira recomendada para gerenciar aplicações Kubernetes em ambiente de produção. Veja a [documentação do Kubectl](https://kubectl.docs.kubernetes.io).
 
 ## Criando objetos
 
-Manifestos Kubernetes podem ser definidos em YAML ou JSON. A extensão de arquivo `.yaml`,
-`.yml`, e `.json` pode ser usado.
+Manifestos Kubernetes podem ser definidos em YAML ou JSON. As extensões de arquivo `.yaml`,
+`.yml`, e `.json` podem ser usadas.
 
 ```bash
-kubectl apply -f ./my-manifest.yaml            # criar recurso(s)
-kubectl apply -f ./my1.yaml -f ./my2.yaml      # criar a partir de vários arquivos
-kubectl apply -f ./dir                         # criar recurso(s) em todos os arquivos de manifesto no diretório
-kubectl apply -f https://git.io/vPieo          # criar recurso(s) a partir de URL
-kubectl create deployment nginx --image=nginx  # iniciar uma única instância do nginx
+kubectl apply -f ./my-manifest.yaml            # cria recurso(s)
+kubectl apply -f ./my1.yaml -f ./my2.yaml      # cria a partir de vários arquivos
+kubectl apply -f ./dir                         # cria recurso(s) em todos os arquivos de manifesto no diretório
+kubectl apply -f https://git.io/vPieo          # cria recurso(s) a partir de URL
+kubectl create deployment nginx --image=nginx  # inicia uma única instância do nginx
 
-# crie um Job que imprime "Hello World"
+# cria um Job que exibe "Hello World"
 kubectl create job hello --image=busybox:1.28 -- echo "Hello World"
 
-# crie um CronJob que imprime "Hello World" a cada minuto
+# cria um CronJob que exibe "Hello World" a cada minuto
 kubectl create cronjob hello --image=busybox:1.28   --schedule="*/1 * * * *" -- echo "Hello World"
 
-kubectl explain pods                           # obtenha a documentação de manifesto do pod
+kubectl explain pods                           # obtem a documentação de manifesto do pod
 
-# Crie vários objetos YAML a partir de stdin
+# Cria vários objetos YAML a partir de stdin
 cat <<EOF | kubectl apply -f -
 apiVersion: v1
 kind: Pod
```

Comment:
Minor: "image: busybox:1.28" instead of "image: busybox"

Reply:
but if I leave it without the tag won´t it assume latest?

Reply:
Yes, you're right! 

I commented because the version is added to the original text.